app/src/main/java/com/example/coinage/fragments/profile/EditInfoFragment.java

```diff
@@ -0,0 +1,104 @@
+package com.example.coinage.fragments.profile;
+
+import android.content.Context;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.TextView;
+
+import com.example.coinage.R;
+import com.parse.ParseUser;
+
+public class EditInfoFragment extends Fragment {
+    public static final String TAG = "EditInfoFragment";
+
+    private TextView tvNameEditDisplay;
+    private TextView tvNameEdit;
+    private ConstraintLayout clEmailEdit;
+    private ConstraintLayout clPasswordEdit;
+
+    public EditInfoFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_edit_info, container, false);
+    }
+
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        tvNameEditDisplay = view.findViewById(R.id.tvNameEditDisplay);
+        tvNameEditDisplay.setText(ParseUser.getCurrentUser().getUsername());
+
+        tvNameEdit = view.findViewById(R.id.tvNameEdit);
+        tvNameEdit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "change name");
```

Comment:
nit - Logs can be as long as you want them, good to remove ambiguity!

Are you notifying the user the name needs to be changed? Are you actually changing the name yet? 
Maybe something like "Opening name editing view"?